packages/node/test/integration/uninstall.spec.ts

```diff
@@ -174,16 +174,23 @@ describe("Node A and B install apps of different outcome types, then uninstall t
       let nodeAProcessID;
       let nodeBProcessID;
 
+      const verifyId = () => {
+        if (nodeAProcessID && nodeBProcessID) {
+          expect(nodeBProcessID).toEqual(nodeAProcessID);
+          done();
+        }
+      };
+
       nodeA.on(NODE_EVENTS.UNINSTALL_FINISHED, async (msg: FinMessage) => {
```

Comment:
is ordering important here? failed the previous test because it emits on nodeB then nodeA. the changes i made the tests un-opinionated on this